api/src/pcapi/core/educational/models.py

```diff
@@ -2050,6 +2051,18 @@ class CollectiveDmsApplication(PcObject, models.Base, models.Model):
     processingDate = sa.Column(sa.DateTime, nullable=True)
     userDeletionDate = sa.Column(sa.DateTime, nullable=True)
 
+    @hybrid_property
+    def siren(self):
+        return self.siret[:9]
```

Comment:
Nitpick, mais comme je le vois utilisé ailleurs:
```suggestion
        return self.siret[:SIREN_LENGTH]
```

Reply:
Bien vu ! Je viens de corriger.